src/win/fs.c

```diff
@@ -1179,7 +1179,7 @@ INLINE static int fs__stat_handle(HANDLE handle, uv_stat_t* statbuf) {
    * Therefore we'll just report a sensible value that's quite commonly okay
    * on modern hardware.
```

Comment:
Maybe mention here the Advanced Format and add a note why this is 4k?

Reply:
Done